one_fm/templates/pages/job_application.js

```diff
@@ -110,7 +110,7 @@ job_application = Class.extend({
   },
   show_cv_section: function(applicant_name) {
     $(".applicant_cv").empty();
-    $(".applicant_cv").append(`<h5>So ${applicant_name}, Attach your CV here to know more about you and then Submit.</h5>`)
+    $(".applicant_cv").append(`<h5>So ${applicant_name}, Attach your CV here to know more about you and then submit it.</h5>`)
```

Comment:
"Dear {applicant}, please attach your CV here so we can get to know you better."